packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -329,7 +329,8 @@ describe('Connection string', function () {
   });
 
   it('can connect to an Atlas replicaset without srv', async function () {
-    if (!hasAtlasEnvironmentVariables()) {
+    if (!hasAtlasEnvironmentVariables() || TEST_COMPASS_WEB) {
```

Comment:
Please don't skip those, most of our web tests will use the user-pass based auth, let's rather exclude sasl prep only for published bundle, and not for one that will be compiled for tests

Reply:
Good call. Updated the webpack to only run on `production` with the comment.